include/z64.h

```diff
@@ -32,6 +32,8 @@
 #include "z64math.h"
 #include "z64map_mark.h"
 #include "z64message.h"
+#include "z64olib.h"
```

Comment:
olib seems to be used in like 6 files. Why not just include the header in those files instead of leaving it here?

Reply:
I find it much easier to cleanup includes per-source-file all at once rather than find all the source files a given header should be included in